filament/include/filament/Material.h

```diff
@@ -87,10 +89,15 @@ class UTILS_PUBLIC Material : public FilamentAPI {
         Builder& operator=(Builder const& rhs) noexcept;
         Builder& operator=(Builder&& rhs) noexcept;
 
-        // This does not copies the content of the RAM, only copy references.
+        // This does not copy the content of the RAM, it only makes a weak reference.
         // The RAM must stay valid until build() is called.
         Builder& package(const void* payload, size_t size);
 
+        // Alternatively, clients can submit a BufferDescriptor instead of void *, which is
+        // auto-released if a callback is present. Note that some language bindings (e.g.,
```

Comment:
I'm not sure I understand. BufferDescriptor does not release the memory, even when a callback is present.